diff --git a/core/dnsserver/zdirectives.go b/core/dnsserver/zdirectives.go
index cf39df3..1fbb04e 100644
--- a/core/dnsserver/zdirectives.go
+++ b/core/dnsserver/zdirectives.go
@@ -51,15 +51,9 @@ var Directives = []string{
 	"template",
 	"transfer",
 	"hosts",
-	"route53",
-	"azure",
-	"clouddns",
-	"k8s_external",
-	"kubernetes",
 	"file",
 	"auto",
 	"secondary",
-	"etcd",
 	"loop",
 	"forward",
 	"grpc",
diff --git a/core/plugin/zplugin.go b/core/plugin/zplugin.go
index 476d8c1..76a5fdc 100644
--- a/core/plugin/zplugin.go
+++ b/core/plugin/zplugin.go
@@ -9,20 +9,17 @@ import (
 	_ "github.com/coredns/coredns/plugin/any"
 	_ "github.com/coredns/coredns/plugin/auto"
 	_ "github.com/coredns/coredns/plugin/autopath"
-	_ "github.com/coredns/coredns/plugin/azure"
 	_ "github.com/coredns/coredns/plugin/bind"
 	_ "github.com/coredns/coredns/plugin/bufsize"
 	_ "github.com/coredns/coredns/plugin/cache"
 	_ "github.com/coredns/coredns/plugin/cancel"
 	_ "github.com/coredns/coredns/plugin/chaos"
-	_ "github.com/coredns/coredns/plugin/clouddns"
 	_ "github.com/coredns/coredns/plugin/debug"
 	_ "github.com/coredns/coredns/plugin/dns64"
 	_ "github.com/coredns/coredns/plugin/dnssec"
 	_ "github.com/coredns/coredns/plugin/dnstap"
 	_ "github.com/coredns/coredns/plugin/erratic"
 	_ "github.com/coredns/coredns/plugin/errors"
-	_ "github.com/coredns/coredns/plugin/etcd"
 	_ "github.com/coredns/coredns/plugin/file"
 	_ "github.com/coredns/coredns/plugin/forward"
 	_ "github.com/coredns/coredns/plugin/geoip"
@@ -33,8 +30,6 @@ import (
 	_ "github.com/coredns/coredns/plugin/hosts"
 	_ "github.com/coredns/coredns/plugin/https"
 	_ "github.com/coredns/coredns/plugin/https3"
-	_ "github.com/coredns/coredns/plugin/k8s_external"
-	_ "github.com/coredns/coredns/plugin/kubernetes"
 	_ "github.com/coredns/coredns/plugin/loadbalance"
 	_ "github.com/coredns/coredns/plugin/local"
 	_ "github.com/coredns/coredns/plugin/log"
@@ -52,7 +47,6 @@ import (
 	_ "github.com/coredns/coredns/plugin/reload"
 	_ "github.com/coredns/coredns/plugin/rewrite"
 	_ "github.com/coredns/coredns/plugin/root"
-	_ "github.com/coredns/coredns/plugin/route53"
 	_ "github.com/coredns/coredns/plugin/secondary"
 	_ "github.com/coredns/coredns/plugin/sign"
 	_ "github.com/coredns/coredns/plugin/template"
diff --git a/plugin.cfg b/plugin.cfg
index a4fc637..9a0c121 100644
--- a/plugin.cfg
+++ b/plugin.cfg
@@ -60,15 +60,9 @@ minimal:minimal
 template:template
 transfer:transfer
 hosts:hosts
-route53:route53
-azure:azure
-clouddns:clouddns
-k8s_external:k8s_external
-kubernetes:kubernetes
 file:file
 auto:auto
 secondary:secondary
-etcd:etcd
 loop:loop
 forward:forward
 grpc:grpc
